### PR TITLE
Report on supported distributions

### PR DESCRIPTION
**Pull request by richfitz**
~Merge after #51, contains those commits.~

A very small helper function that we need for odin2 to know what distribution functions it can use.

### CODE REVIEW

File: R/dsl-distributions.R [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+##' information about which distributions and arguments would succeed
+##' there.
+##'
+##' @title Information about supported distributions
+##'
+##' @return A data.frame with columns
+##'
+##' * `name` the name of the distribution; each name begins with a
+##'   capital letter, and there are duplicate names where different
+##'   parametrisations are supported.
```
Comment (weshinsley) [inline R/dsl-distributions.R:13]:
```suggestion
##'   parameterisations are supported.
```

File: R/dsl-distributions.R [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+##' * `args` the arguments of all parameters, *except* the random
+##'   variable itself which is given as the first argument to density
+##'   functions.
+##'
+##' We may expand the output here in future to include information on
```
Comment (weshinsley) [inline R/dsl-distributions.R:18]:
```suggestion
##' We may expand the output here in the future to include information on
```

Comment (weshinsley) [reply R/dsl-distributions.R:18]: You quite often type "in future" without "the" - so perhaps this is a dialect issue!

**Review by weshinsley — APPROVED** [review]
Easy one!

File: R/dsl-distributions.R (hunk 1 above)
Comment (richfitz) [reply R/dsl-distributions.R:13]: hah, I googled that and found *four* variants. I don't have strong feelings though

File: R/dsl-distributions.R (hunk 2 above)
Comment (richfitz) [reply R/dsl-distributions.R]: yeah, "in future" reads fine to me, but I can see that "in the future" is probably more Proper

**Pull request merged**